references/cli/inferless-model.mdx

```diff
@@ -9,34 +9,41 @@ You can use this to manage your models.
 - `list`: List all models in the current workspace
 - `delete`: delete a model from the system.
 - `rebuild`: rebuild a model.
-- `info`: Get model details  (min replicas, max replicas, current replicas, status).
+- `info`: Get model details (min replicas, max replicas, current replicas, status).
 - `activate`: activate a model this will restore the min and max replicas to the original values.
-- `deactivate`: deactivate a model this will scale the min and max replicas to 0. 
+- `deactivate`: deactivate a model this will scale the min and max replicas to 0.
+- `patch`: patch model configuration.
 
 ### Example
 
 Below command displays all the models in the workspace with their details.
 
 ```bash
-inferless model list 
+inferless model list
 ```
 
 Below command rebuilds a model.
 
 ```bash
-inferless model rebuild --model-id <model_id> 
+inferless model rebuild --model-id <model_id>
```

Comment:
Can you also add an example for runtime + code change

Reply:
I have added this in the next line. inferless model rebuild --local